[2021-12-21 20:38:01,970] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task2 2021-12-20T00:00:00+00:00 [queued]>
[2021-12-21 20:38:02,005] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task2 2021-12-20T00:00:00+00:00 [queued]>
[2021-12-21 20:38:02,005] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 20:38:02,005] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2021-12-21 20:38:02,005] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 20:38:02,041] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task2> on 2021-12-20T00:00:00+00:00
[2021-12-21 20:38:02,053] {standard_task_runner.py:52} INFO - Started process 72811 to run task
[2021-12-21 20:38:02,089] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task2', '2021-12-20T00:00:00+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmplycye10v', '--error-file', '/tmp/tmpu4z9hh0l']
[2021-12-21 20:38:02,089] {standard_task_runner.py:77} INFO - Job 10: Subtask task2
[2021-12-21 20:38:02,293] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task2 2021-12-20T00:00:00+00:00 [running]> on host airflowvm
[2021-12-21 20:38:02,533] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task2
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:00:00+00:00
[2021-12-21 20:38:02,533] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 20:38:02,533] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-12-21 20:38:02,549] {subprocess.py:75} INFO - Output:
[2021-12-21 20:38:05,559] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-21 20:38:05,656] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task2, execution_date=20211220T000000, start_date=20211221T203801, end_date=20211221T203805
[2021-12-21 20:38:05,732] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-21 20:38:05,743] {local_task_job.py:151} INFO - Task exited with return code 0
